<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSendDialog</class>
 <widget class="QDialog" name="CSendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_model">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>模式查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_model">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_model">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>停止模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>采集模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>工程模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>校准模式</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_setmodel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>模式设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>模式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>122</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>14H:通道</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_14H_channel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_14H_LED">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>熄灭</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>点亮</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_14H_set">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>14H_设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32H_12H">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32H_14H">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_adjust_15H">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>校准参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32H_15H">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_version">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>固件查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_version">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
